### ISSUE TITLE
Corrupt SUNDIALSConfig.cmake generated if SUNDIALS_BUILD_WITH_PROFILING is off

### ISSUE THREAD

**Issue by antonio-rojas** [LLNL/sundials#102]
When `SUNDIALS_BUILD_WITH_PROFILING` is off, the variable `ENABLE_CALIPER` is marked as internal, which makes cmake generate an unusable SUNDIALSConfig.cmake with the following broken code:
```
if( AND NOT TARGET caliper)
  find_package(CALIPER PATHS )
endif()
```

**Comment by balos1** [comment]
Fixed in SUNDIALS v6.1.0. Thanks!

**Comment by afchadwi** [comment]
Hi, I just tried installing v6.1.0 with `SUNDIALS_BUILD_WITH_PROFILING` set to off, both from tarball and via Spack (manually checksummed), and the SUNDIALSConfig.cmake that was generated still seems to contain the reference to try and find Caliper.

However, manually deleting the three lines that check for Caliper seems to fix it, and everything seems to compile and execute correctly.

I don't know if this has an effect, but this was on an M1 MacBook Air with MacOS 12.1 and the Apple command line tools with gfortran 11.2.0, cmake 3.22.1, and openmpi 4.1.2 all installed from Homebrew. The only non-standard option I had turned on for Sundials was the F2003 bindings, but running `make test` after building didn't show any failures.
